content/docs/reference-events.md

```diff
@@ -93,13 +93,13 @@ The event handlers below are triggered by an event in the bubbling phase. To reg
 
```

Comment:
## Referensi (#reference)

What do you think? 😄

Reply:
:+1: